android/src/main/res/layout/default_media_controller.xml

```diff
@@ -8,7 +8,7 @@
     android:background="@color/bmc_background"
     android:layout_width="match_parent"
     android:layout_height="@dimen/bc_control_bar_height"
-    android:padding="8dp"
+    android:padding="@dimen/bc_control_bar_padding"
```

Comment:
Not sure about best practices in android, but wouldn't be easier to read if you just put `2dp` instead of pointing to a variable on a separate file?

Reply:
Hey @rafbc , good point, actually yes this is the best practice in Android; to place all dimension values and hardcoded strings in separate XML files. This makes it easier to change the value from one place and apply it to all usages. For instance, I am using this value in both default_media_controller.xml files (layout and layout-sw480). And let us say that we decided to have layout-sw480 use a different value of the same variable, we can then override the default file dimens.xml file (located in values folder) inside a folder called values-sw480.

Regarding readability, when using Android Studio, it automatically places a placeholder with the value if the variable where it is used in the XML file, like in Android Studio you will see `android:padding="2dp"` `2dp` will be greyish, and when you click on it, it reveals `@dimen/bc_control_bar_padding`, then command-click on it, takes to the declaration file to edit it.
Also it is worth mentioning that if you have lint enabled in Android Studio, and you use hardcoded values/strings in the layout file, it marks them with a warning to move them to the dimesn/strings files.